src/amt/models.ts

```diff
@@ -327,9 +331,9 @@ export namespace Models {
     // If the value of this read-only field is True, the value of AcceptNonSecureConnections can be changed. Note that this class and field can be accessed locally as well as remotely.
   }
 
-  export interface GenerateKeyPair {
-    KeyAlgorithm: Types.GenerateKeyPair.KeyAlgorithm
-    KeyLength: number
+  export interface KeyPairParameters {
+    KeyAlgorithm: Types.KeyPairParameters.KeyAlgorithm
```

Comment:
Name redundancy ... could KeyAlgorithnm just be Algorithm? (same for Length)

Reply:
KeyAlgorithm is a named property in the AMT SDK, so for things like that I'm trying to be consistent so that the AMT SDK can easily be used as a reference for looking up what a property is and how it is used.

I did have to get creative on this interface and named it KeyPairParameters as GenerateKeyPair didn't really fit well as that is a function name and not an actual model and these are parameters that feed into the GenerateKeyPair function

Reply:
Updated the Types and Interface naming to be GenerateKeyPairParameters.  This aligns the naming for the parameters to closely resemble the function in which they are used.